EIPS/eip-663.md

```diff
@@ -72,14 +86,14 @@ This has no effect on backwards compatibility because the opcodes were not previ
 Given variable `n`, which equals to `imm + 1`, for `1 <= n <= 256`:
 
     - `DUPN imm` to fail validation if `stack_height < n`.
-    - `SWAPN imm` to fail validation if `stack_height < (n + 1)`.
+    - `EXCHANGE imm` to fail validation if `stack_height < `.
```

Comment:
need upper range.  n+m?

Reply:
added in https://github.com/ethereum/EIPs/pull/7819/commits/e28a856469803c11b0f41f58d1e8bd2dbb1fe71a